pbj-core/buildSrc/src/main/kotlin/com.hedera.pbj.root.gradle.kts

```diff
@@ -20,6 +39,26 @@ plugins {
     id("com.hedera.pbj.spotless-conventions")
     id("com.hedera.pbj.spotless-kotlin-conventions")
     id("com.autonomousapps.dependency-analysis")
+    id("io.github.gradle-nexus.publish-plugin")
 }
 
+group = "com.hedera.pbj"
+
 spotless { kotlinGradle { target("buildSrc/**/*.gradle.kts") } }
+
+nexusPublishing {
+    repositories {
+        sonatype {
+            username = System.getenv("OSSRH_USERNAME")
+            password = System.getenv("OSSRH_PASSWORD")
+        }
+    }
+}
+
+tasks.withType<CloseNexusStagingRepository>().configureEach {
+    mustRunAfter(tasks.withType<PublishToMavenRepository>())
+}
+
+tasks.withType<ReleaseNexusStagingRepository>().configureEach {
+    mustRunAfter(tasks.withType<PublishToMavenRepository>())
+}
```

Comment:
These have no effect as the `withType` only is about tasks in the current project – _root_ in this case. I think we should use `dependsOn` a bit differently. But it is a flaw in the nexus plugin (IMO) that things are split between root and subprojects in the way they are.